.github/workflows/release.yml

```diff
@@ -0,0 +1,97 @@
+name: Create Python wheels
+
+on:
+  push:
+    branches:
+      - "release/**"
+  # REMOVE BEFORE MERGING to only run on pushes to a release branch:
+  pull_request:
+
+jobs:
+  python-wheel-mac:
+    name: Create Python wheel for macOS
+    runs-on: macos-10.15
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+
+      - run: make wheel SYMBOLIC_PYTHON=python2
+
+      - uses: actions/setup-node@v1
+
+      - name: Upload to Zeus
+        env:
+          ZEUS_HOOK_BASE: ${{ secrets.ZEUS_HOOK_BASE }}
+        run: |
+          npm install -D @zeus-ci/cli
+          npx zeus job update --build ${{ github.run_id }} --job ${{ github.job }} --ref ${{ github.sha }}
+          npx zeus upload --build ${{ github.run_id }} --job ${{ github.job }} --type "application/zip+wheel" py/dist/*
+          npx zeus job update --build ${{ github.run_id }} --job ${{ github.job }} --status passed
+
+  python-wheel-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        build-arch: [i686, x86_64]
+
+    name: Create Python wheel for Linux ${{ matrix.build-arch }}
```

Comment:
Can we give these jobs a shorter name? For example "Python Linux ${{ matrix.build-arch }}"

https://github.com/getsentry/relay/blob/e5c85a6b83b309f13d598b1e34816c13bbb3f970/.github/workflows/build_library.yml#L15